apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -84,6 +85,10 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
         : null
   );
 
+  const selectedSectionCourseName = selectedSection?.courseDisplayName;
+
+  const showNoUnitAssigned = !selectedSection?.unitId;
```

Comment:
Optional nit: Since both of these variables are only used once, maybe we don't need to define them:
e.g. `showNoUnitAssigned={!selectedSection?.unitId}`